elastic/kibana
# [TSVB] Support triggers only for timeseries chart

## Summary

Fixes https://github.com/elastic/kibana/issues/100064#issuecomment-843958891 
Currently, TSVB reports supported triggers (brush and filter) for **all** charts. But only the timeseries chart actually support them (by clicking and brushing on the chart). 
As a result, when a non-timeseries TSVB chart is embedded on a dashboard, the drilldowns are enabled but they are not working. This causes confusion to our users.
This PR reports the supported triggers only for timeseries.

### How to test it
1. Create a timeseries TSVB chart and embed it to a dashboard. You should be able to create a drilldown
2. Create a non-timeseries TSVB chart (for example TopN) and embed it to a dashboard. You shouldn't be able to create a drilldown.

![image](https://user-images.githubusercontent.com/17003240/118814572-f7740080-b8b8-11eb-83d7-b73bca051130.png)
![image](https://user-images.githubusercontent.com/17003240/118814630-065ab300-b8b9-11eb-8db0-ced2363c32e6.png)
